collections/_infographics/policies/fit-for-55.md

```diff
@@ -1,13 +1,134 @@
+caption:       "'Fit for 55' is a package of legislation proposals designed by the European Commission to cut the EU's net greenhouse gas emissions by 55% below 1990 levels by 2030. It is a key step for the EU on its way to reach its legally binding target – net zero by 2050."
 ---
 
-{% include includes-local/comment-placeholder.md slug="infografiky/fit-for-55" %}
+{% comment %}
+{% include preview-box.html
+    title="What are the individual measures in the 'Fit for 55' package?"
+    text="In the related infographic, we briefly discuss the content of the individual legislative proposals in the package."
+    slug="fit-for-55-opatreni"
+%}
+{% endcomment %}
+
+['Fit for 55'](https://eur-lex.europa.eu/legal-content/en/TXT/?uri=CELEX%3A52021DC0550), a package of legislative proposals that promotes a **reduction of greenhouse gas emissions by 55% by 2030 compared to 1990**, was presented by the European Commission in July 2021. The package includes proposals amending existing directives or regulations and entirely new legislative proposals, focusing on emissions trading (EU ETS), carbon offsetting at borders, energy and fuels, and land use and forestry. Last but not least, it brings in supporting measures to make the transition more socially just. The proposed measures should bring new opportunities for investment, innovation and job creation[^fit-for-55].
+
+The package is currently at the stage of legislative proposals by the European Commission, which are being further discussed in the European Parliament and the Council of the European Union, i.e. with the participation of ministers of the governments of the Member States and national working groups. In this context, the Czech Republic's **presidency of the Council of the EU**,[^presidency-eu], which it will assume in the second half of 2022, is crucial and will thus have the opportunity to influence the direction of the negotiations. It is expected that negotiations on the final form of the legislation could last until 2024.[^fit-for-55-q-a]
+
+## Context of origin
+
+'Fit for 55' is part of the European Union's (EU) broader strategy to achieve  **<glossary id="carbon-neutrality">Carbon-neutrality</glossary>** by 2050, known as the [**European Green Deal**](https://ec.europa.eu/info/strategy/priorities-2019-2024/european-green-deal).  The European Commission presented the Green Deal in 2019, after which member states agreed on it in 2020. Then, in the summer of 2021, the [**European Climate Law Framework**](https://eur-lex.europa.eu/legal-content/en/TXT/?uri=CELEX%3A32021R1119) came into force, making the achievement of carbon neutrality by 2050 legally binding. This law includes a mid-term target that commits the EU to a 55% reduction in greenhouse gas emissions by 2030 compared to 1990, which the 'Fit for 55' package (loosely translated as "ready for 55%") is designed to ensure is achieved.
```

Comment:
- in the written text, "carbon neutrality" shouldn't have a hyphen, not sure how that affects the glossary entry. 
- "which the 'Fit for 55' package (loosely translated as "ready for 55%") is designed to ensure." (remove is achieved)

Reply:
> - in the written text, "carbon neutrality" shouldn't have a hyphen, not sure how that affects the glossary entry.
I fixed the text here and I also had a look at the glossary entry. It had the same mistake so I fixed it there as well.

> - "which the 'Fit for 55' package (loosely translated as "ready for 55%") is designed to ensure." (remove is achieved)
Corrected this text (i.e., removed 'is achieved')